webapp/src/components/user_attribute/user_attribute.jsx

```diff
@@ -23,9 +24,9 @@ export default class UserAttribute extends React.PureComponent {
     }
 
     render() {
-        const {attributes} = this.props;
+        const {attributes, postIsWebhook} = this.props;
 
-        if (attributes == null || attributes.length === 0) {
+        if (attributes == null || attributes.length === 0 || postIsWebhook === 'true') {
```

Comment:
Does it make sense to move the string compare into `index.js` and have `postIsWebhook` be a boolean?

Reply:
+1 on this.

Reply:
Sure! Addressed in the next commit :)